packages/react-experiments/.vscode/settings.json

```diff
@@ -6,7 +6,7 @@
   // When enabled, will trim trailing whitespace when you save a file.
   "files.trimTrailingWhitespace": true,
   // Controls if the editor should automatically close brackets after opening them
-  "editor.autoClosingBrackets": false,
+  "editor.autoClosingBrackets": "never",
```

Comment:
please undo this changes as they are unrelated to this PR scope. ty

Reply:
reverted the changes. thanks for pointing it out.